iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -77,7 +77,7 @@ Fact.prototype.readableValue = function() {
             formattedNumber = "nil";
         }
         else if (d === undefined) {
-            formattedNumber = v;
+            formattedNumber = formatNumber(v,0);
```

Comment:
Add a test [like this](https://github.com/Workiva/ixbrl-viewer/blob/6e57b52f426560a29c2d4a586cecb03f2c56a5d8/iXBRLViewerPlugin/viewer/src/js/fact.test.js#L122-L139) but with a large integral part [like this](https://github.com/Workiva/ixbrl-viewer/blob/6e57b52f426560a29c2d4a586cecb03f2c56a5d8/iXBRLViewerPlugin/viewer/src/js/fact.test.js#L102-L120).

As-is, it looks like this change incorrectly formats INF `1234.56` as `1,234` rather than `1,234.56`.

Reply:
Thanks Brett :). First ever javascript experience, fumbling my way through this.